src/test/java/it/okkam/validation/FiscalCodeValidatorTest.java

```diff
@@ -78,65 +81,65 @@ public void testDateTimePattern() {
         "1926-03-24T00:00:00", "SANGIANO", "M");
     Assert.assertTrue(arrayContains(codes, "FOXDRA26C24H872Y"));
   }
-  
+
   @Test
   public void testDot() {
     String[] codes = FiscalCodeValidator.calcoloCodiceFiscale(conf2, "FO", "DAR.IO",
         "1926-03-24T00:00:00", "SANGIANO", "M");
     Assert.assertTrue(arrayContains(codes, "FOXDRA26C24H872Y"));
   }
-  
+
   @Test
   public void testBacktick() {
     String[] codes = FiscalCodeValidator.calcoloCodiceFiscale(conf2, "D`AMICO", "ILARIA",
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "ILARIA" 11 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS1192)